django/db/backends/sqlite3/schema.py

```diff
@@ -47,6 +47,12 @@ def _remake_table(self, model, create_fields=[], delete_fields=[], alter_fields=
                       override_indexes=None):
         """
         Shortcut to transform a model from old_model into new_model
+
```

Comment:
Yeah, I see why this might have come up in writing tests for this PR, but I think it'll be clearer if that gets merged separately.